### DISCUSSION TITLE
Systemvariablen im Energy Dashboard nutzen?

### DISCUSSION THREAD

**Discussion by fteheesen** [SukramJ/aiohomematic#1318] [Q&A] [answered]
Ich lese die Zählerstände vom Stromzähler per Poweropti-Powerfox (https://poweropti.powerfox.energy/) regelmäßig aus und speichere sie in Homematic-Systemvariablen vom Typ Zahl. Das erfolgt per Node-Red-Flow auf der CCU3.
Die autom. erzeugten Attribute der Systemvariablen-Entität lauten:
		state_class: measurement
		name: <Name der Systemvariablen>
		unit_of_measurement: W
		friendly_name: <Name der Systemvariablen>

Die Integration ins Energie-Dashboard erfordert aber bestimmte Entity-Attribute. Hier ein Beispiel von AVM-DECT200-Schaltern:
(vgl. https://www.home-assistant.io/docs/energy/faq/#troubleshooting-missing-entities)
		state_class: total_increasing
		unit_of_measurement: kWh
		device_class: energy
		friendly_name: <Name der Entität>
Das bedeutet das Attribute device_class fehlt und state_class ist falsch besetzt.
Wenn ich die Attribute mit Entwicklerwerkzeuge: Zustände verändere werden sie beim nächsten Auslesen der Systemvariablen der CCU wieder überschrieben.
Hat jemand eine Idee?
Muss ich evtl. Template-Sensoren erstellen?!

MfG Frank

**Comment by SukramJ** [comment, marked as answer]
Hat eigentlich nichts mit der Integration zu tun, aber geht mit [customize](https://www.home-assistant.io/docs/configuration/customizing-devices/).

**Reply by fteheesen** [reply]
Vielen Dank für den Tip!
